scripts/build-and-pack.sh

```diff
@@ -9,19 +9,29 @@ SCRIPT_DIR=$(dirname $(readlink -f $0))
 # Get the directory of the project.
 PROJECT_BASE_DIR=$(dirname $SCRIPT_DIR)
 
-# Change to the project directory.
-cd $PROJECT_BASE_DIR
-
-# Install dependencies, build, and test.
-echo "npm clean install"
-npm ci
-
-echo "npm run build"
-npm run build
-
-echo "npm run test"
-npm run test
-
-# Create a tarball.
-echo "npm pack"
-npm pack
+# Define the SDK directory.
+SDK_DIR="$PROJECT_BASE_DIR/sdk"
+
+# Loop through each package in the sdk folder.
+for PACKAGE_DIR in "$SDK_DIR"/*; do
```

Comment:
this script is all-in-one. Suppose you will have multiple packages later, do you want to create RC bits and release for all of them everytime? If so, that's fine. If you will release one of these packages alone, I would suggest you accept package_dir as a parameter of this script.